CMakeLists.txt

```diff
@@ -350,7 +350,7 @@ else()
     set(WARNINGS_AS_ERRORS_FLAG "-Werror")
   endif()
   if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
-    set(WARNINGS "${WARNINGS} -Wno-deprecated-register -Wno-error=mismatched-tags -Wno-error=null-conversion -Wno-overloaded-shift-op-parentheses -Wno-error=shift-count-overflow -Wno-error=tautological-constant-out-of-range-compare -Wno-error=unused-private-field -Wno-error=unneeded-internal-declaration")
+    set(WARNINGS "${WARNINGS} -Wno-deprecated-register -Wno-overloaded-shift-op-parentheses -Wno-missing-braces")
```

Comment:
`missing-braces` was triggering lots of warnings from `rctTypes.h` but the code is ok.